ext/rugged/rugged.h

```diff
@@ -147,6 +147,12 @@ struct rugged_cb_payload
     int exception;
 };
 
+struct rugged_treecount_cb_payload
+{
+	unsigned int count;
+	unsigned int limit;
```

Comment:
I just know in libgit2 we used to avoid it in public headers because they way you include the typedef for it can vary from platform to platform. (Interestingly, it looks like it had recently crept in by way of the new `include/git2/sys/stream.h` header.)